test/e2e/page-objects/flows/network.flow.ts

```diff
@@ -40,6 +40,7 @@ export const searchAndSwitchToNetworkFromGlobalMenuFlow = async (
 export const switchToNetworkFromSendFlow = async (
   driver: Driver,
   networkName: string,
+  tokenSymbol = 'ETH',
```

Comment:
we default to ETH as it's the token ticker in most cases (and in all cases they use this flow)